### ISSUE TITLE
update services endpoint

### ISSUE THREAD

**Issue by tomkralidis** [geopython/demo.pycsw.org#2]
As per https://lists.osgeo.org/pipermail/pycsw-devel/2019-January/000805.html, the services in 
http://demo.pycsw.org/services/csw?service=CSW&version=2.0.2&request=GetCapabilities are outdated/stale.  Updated the endpoints and add a text file of an endpoint list that can be regenerated as required.

**Comment by jmckenna** [comment]
I agree that a text file explaining the endpoints would be very useful.
